### PR TITLE
Remove remaining GLisms from command tree

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

**Comment by chrisforbes** [comment]
Vulkan draw call grouping works like debug marker or renderpass grouping.

**Comment by stellama0208** [comment]
If the draw call grouping toggle should be kept, a check for `p.GroupByDrawCall` at marker stack pushing/popping will do it.

File: cmd/gapit/sxs_video.go [hunk 1]
```diff
@@ -101,23 +99,10 @@ func (verb *videoVerb) sxsVideoSource(
...
 	frameIndex, numDrawCalls := 0, 0
 
-	// Some traces call eglSwapBuffers without actually drawing to or clearing
-	// the framebuffer. This is most common during loading screens.
-	// These would result in a failed comparison as the observed frame could
-	// be anything and the replayed frame will show the undefined framebuffer
-	// pattern.
-	// Permit the first run of frames to have no content. If there are no
-	// draw-calls or clear calls at all however, then do not permit this.
+	// TODO: in GLES we used to handle empty frames specially here. Work out
```
Comment (pau-baiget) [inline cmd/gapit/sxs_video.go:102]: Please file a bug for this so we don't forget, thanks!

File: gapis/resolve/stats.go [hunk 2]
```diff
@@ -90,17 +90,14 @@ func drawCallStats(ctx context.Context, capt *path.Capture, stats *service.Stats
 			if err != nil {
 				return err
 			}
-			// If the command has subcommands, ignore it (vkQueueSubmit or similar)
```
Comment (pau-baiget) [inline gapis/resolve/stats.go:93]: Isn't this comment still relevant?

File: gapis/resolve/events.go [hunk 3]
```diff
@@ -84,35 +84,13 @@ func Events(ctx context.Context, p *path.Events, r *path.ResolveConfig) (*servic
...
 			pending = append(pending, service.EventKind_FirstInFrame)
 		}
-		if p.Clears && f.IsClear() {
-			events = append(events, &service.Event{
-				Kind:      service.EventKind_Clear,
-				Command:   p.Capture.Command(uint64(id)),
-				Timestamp: getTime(cmd),
-			})
-		}
-		// Add all non-special event types
```
Comment (pau-baiget) [inline gapis/resolve/events.go:94]: Isn't this comment still relevant for the surviving types?

File: gapis/resolve/stats.go (hunk 2 above)
Comment (chrisforbes) [reply gapis/resolve/stats.go:93]: No. The code below handles subcommands; the comment is just wrong.

**Pull request closed without merging**